### PR TITLE
Feature/add welcome status section

### LINKED ISSUE #26: Create a functional component thatt displays a message to the user.
When the user logs in, the user can appreciate a welcoming message

### PR DESCRIPTION
**Pull request by Atos20**
#### What’s this PR do?  
- It creates a functional component called MovieSection that display a welcoming message
- It styles the movie section 
#### Where should the reviewer start?  

 `MovieSection.js` and `MovieSection.scss`

#### How should this be manually tested?  
 run `npm start`
#### Any background context you want to provide?  

-  This is just a functional component that should only update the name of the user.
-  I think we can make use of the component to display just the name of the user that logs into the application.

#### What are the relevant tickets?  
 closes #26 

#### Screenshots (if appropriate) 
Screen Shot 2020-10-06 at 11 58 48 PM

#### Questions: 
Now that I think of, I should have named the movie section => welcoming section or something like that

### CODE REVIEW

**Comment by Atos20** [comment]
Sorry :P I misspelled your last name !!

**Comment by Atos20** [comment]
I went ahead and rename the folder from `movieSection` to `welcomingSection`

File: src/App/App.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import React, {Component} from 'react';
 import {LoginForm} from '../LoginForm/LoginForm'
```
Comment (holladayian) [inline src/App/App.js:2]: Minor cleanup: 
```
{Component} => { Component }
{LoginForm} => { LoginForm }
```

File: src/App/App.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
...
 import { Homepage } from '../Homepage/Homepage'
+import { MovieSection } from '../MovieSection/MovieSection'
 import './App.scss';
 
 class App extends Component{
```
Comment (holladayian) [inline src/App/App.js:7]: Minor cleanup: 
```
Component{ => Component {
```

File: src/MovieSection/MovieSection.scss [hunk 3]
```diff
@@ -9,19 +9,40 @@
     // background-position: 50px 170em;
     // opacity: 0.9;
     background-repeat: no-repeat;
+    padding: 2%;
+
+    .user-welcome{
```
Comment (holladayian) [inline src/MovieSection/MovieSection.scss]: I think for CSS style guide we want a space between the class property and opening curly brace
```
.user-welcome {
```

**Comment by holladayian** [comment]
Looks good to me! Just some minor comments on style guide cleanup. I'll go a head and merge.

**Pull request merged**